package.json

```diff
@@ -85,7 +86,8 @@
     "electron-to-chromium": "^1.4.212",
     "enquirer": "^2.3.6",
     "esbuild": "^0.18.11",
-    "eslint": "^8.37.0",
+    "eslint": "^8.55.0",
+    "eslint-plugin-internal-playwright": "file:utils/eslint-plugin-internal-playwright",
```

Comment:
Note: According to my research there seems to be no other way of consuming an ESLint rule without treating it as a local package.